ethereumj-core/src/main/java/org/ethereum/jsonrpc/JsonRpc.java

```diff
@@ -36,6 +36,7 @@ public String toString() {
         public String gasPrice;
         public String value;
         public String data; // compiledCode
+        public String nonce;
```

Comment:
What is the meaning of `nonce` here? I couldn't find it in the JSON-RPC docs: https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_call